gantry/clients/__init__.py

```diff
@@ -0,0 +1,3 @@
+# flake8: noqa
+from .gitlab import GitlabClient
+from .prometheus import PrometheusClient
```

Comment:
I think it'd actually be okay to not import the clients here and require an extra level of indirection when importing them later on. That way it's possible to import one without requiring us to load the other.